File: Python/Algorithms/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums):
        max_, count = 0, 0
        for i in nums:
            if i:
                count = count + 1
                if count > max_:
                    max_ = count
                continue
            count = 0
        return max_

File: Rust/Algorithms/485.rs
impl Solution {
    pub fn find_max_consecutive_ones(nums: Vec<i32>) -> i32 {
        let (mut max, mut count) = (0, 0);
        for i in nums {
            if (i == 1) {
                count += 1;
                if (count > max) {
                    max = count;
                }
                continue;
            }
            count = 0;
        }
        max
    }
}
